src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx

```diff
@@ -309,12 +317,20 @@ const ProjectSelector = createReactClass({
 
   render() {
     let org = this.props.organization;
+    let features = new Set(org.features);
     let access = new Set(org.access);
     let hasSingleTeam = org.teams.length === 1;
 
-    let projectList = sortArray(this.state.projectList, ([team, project]) => {
-      return [!project.isBookmarked, team.name, project.name];
-    });
+    let projectList;
+    if (features.has('internal-catchall')) {
```

Comment:
i would say something abt duplicating the logic in a slightly nicer way, but i guess this temporary code anyway

Reply:
yeah the pr i merged yesterday also has a decent amount of this grossness. i couldn't come up with anything that clever and figure it should only be around for a month or so, but if you have suggestions, i'm down!